src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingTable.js

```diff
@@ -243,7 +243,12 @@ const BenchmarkingTable = (props) => {
                 onResetSelectedItems,
             }}
             columnExtensionMap={benchmarkingColExtMap}
-            hideFacets={['dataset']}
+            hideFacets={[
+                'dataset',
+                'file_sets.libraries.analytes.samples.sample_sources.code',
+                'status',
+                'validation-errors',
```

Comment:
This doesn't seem to be doing the job. Not sure why, I'll need to look into it more to determine why this facet is still showing up on staging.